erpnext/accounts/doctype/account/chart_of_accounts/ni_ni_chart_template.json

```diff
@@ -0,0 +1,237 @@
+                "Venta de Bienes o Prestacion de Servicios Exonerados": {},
+                "Venta por Exportaciones": {}
+            },
+            "OTROS INGRESOS GRABALES": {},
+            "INGRESOS NO GRABALES": {},
+            "root_type": "Income"
+        },
+        "PASIVO": {
+            "PASIVO CORRIENTE": {
+                "CUENTAS POR PAGAR A PROVEEDORES": {
+                    "Cuentas por Pagar Moneda Nacional": {},
+                    "Cuentas por Pagar Moneda Extrangera": {},
+                    "Documentos por Pagar Moneda Nacional": {},
+                    "Documentos por Pagar Moneda Extrangera": {},
+                    "Proveedores por Importaciones": {}
+                },
+                "ANTICIPOS DE CLIENTES": {},
+                "DEUDAS FINANCIERAS": {
+                    "Prestamos por Pagar a Corto Plazo": {},
+                    "Otras Deudas Bancarios": {}
+                },
+                "GASTOS POR PAGAR": {
+                    "Servicios Basicos": {},
+                    "Prestaciones Sociales": {},
+                    "Salarios por Pagar": {}
+                },
+                "PROVISIONES POR PAGAR": {
+                    "Pasivos Laborales": {},
+                    "Reclamos por Pagar": {},
+                    "Responsabilidad frente a terceros": {}
+                },
+                "IMPUESTOS POR PAGAR": {
+                    "Impuesto al Valor Agregado por Pagar": {},
+                    "Impuesto sobre la Renta": {},
+                    "Impuestos Municipales": {}
+                },
+                "RETENCIONES POR PAGAR": {
+                    "Rentas del Trabajo": {},
+                    "Rentas de Actividades Economicas": {},
+                    "Rentas y Ganancias de Capital": {}
+                },
+                "OTRAS CUENTAS POR PAGAR": {}
+            },
+            "PASIVO NO CORRIENTE": {
+                "PRESTAMOS A LARGO PLAZO": {},
+                "DEUDAS COMERCIALES": {},
+                "DEUDAS DIVERSAS": {},
+                "DEUDAS FINANCIERAS": {}
+            },
+            "root_type": "Liability"
+        },
+        "PATRIMONIO": {
+            "APORTE DE PROPIETARIOS/SOCIOS": {
+                "APORTES A CAPITALIZAR": {},
+                "CAPITAL": {
+                    "Capital Social": {
+                        "account_type": "Equity"
+                    }
+                }
+            },
+            "DONACIONES": {},
+            "GANANCIAS RETENIDAS": {
+                "RESERVAS": {
+                    "Reservas Legales": {
+                        "account_type": "Equity"
+                    },
+                    "Reservas Voluntarias": {
+                        "account_type": "Equity"
+                    }
+                },
+                "RESULTADOS ACUMULADOS": {
+                     "Resultados Acumulados": {
+                        "account_type": "Equity"
+                    }
```

Comment:
As style guide, always best to use Title Case eg: **Donaciones** instead of **DONACIONES**